pintracker/stateless/stateless.go

```diff
@@ -542,7 +444,7 @@ func (spt *Tracker) ipfsStatusAll(ctx context.Context) (map[string]*api.PinInfo,
 	ctx, span := trace.StartSpan(ctx, "tracker/stateless/ipfsStatusAll")
 	defer span.End()
 
-	var ipsMap map[string]api.IPFSPinStatus
+	ipsMap := make(map[string]api.IPFSPinStatus)
```

Comment:
why are you allocating a map which is immediately going to be discarded by setting the pointer to the map provided by the IPFSConnector